pkg/cmd/roachtest/tpchbench.go

```diff
@@ -114,6 +117,44 @@ func runTPCHBench(ctx context.Context, t *test, c *cluster, b tpchBenchSpec) {
 	m.Wait()
 }
 
+// getNumQueriesInFile downloads a file that url points to, stores it at
+// filename, parses it using querybench, and deletes the file. It returns the
+// number of queries in the file.
+func getNumQueriesInFile(filename, url string) (int, error) {
+	if err := downloadFile(filename, url); err != nil {
+		return 0, err
+	}
+	queries, err := querybench.GetQueries(filename)
+	if err != nil {
+		return 0, err
+	}
+	if err = os.Remove(filename); err != nil {
+		return 0, err
+	}
+	return len(queries), nil
+}
+
+// downloadFile will download a url to a local file.
+func downloadFile(filepath string, url string) error {
```

Comment:
Do you want to put this file in the `/tmp` directory? See [`ioutil.TempFile`](https://golang.org/pkg/io/ioutil/#TempFile).